src/lib/FileManager.js

```diff
@@ -105,7 +105,13 @@ export default class FileManager {
   }
 
   async downloadFileFromDescriptor(fileDescriptor) {
+    var retries = 3;
```

Comment:
Should especially be const since we never want this number to change.

Reply:
Well actually I guess you're decrementing this in the while loop so nevermind..

Reply:
I will change this to es6 in the beautify-code branch.